### DISCUSSION TITLE
Disable git blame above current line

### DISCUSSION THREAD

**Discussion by lebzm** [gitkraken/vscode-gitlens#1513] [Q&A]
I like the git blame at the end of the current line but want to disable git blame inside the yellow box, how to do it? I know how to disable the current line one with `"gitlens.currentLine.enabled": false,` but can't find the setting for the other one.

![Screenshot from 2021-05-19 13-38-18](https://user-images.githubusercontent.com/20436235/118767109-88da7700-b8a7-11eb-8b4f-9c5757042320.png)

**Comment by eamodio** [comment]
Those annotations in VS Code are called Code Lens, and the setting in GitLens to turn those off completely is `"gitlens.codeLens.enabled": false`. You can also much more easily see and explorer GitLens' settings, by opening the GitLens interactive settings editor from the command palette (`ctrl+shift+p` or `cmd+shift+p`) and search for _GitLens: Open Settings_

![image](https://user-images.githubusercontent.com/641685/119272756-8ba1d880-bbd5-11eb-9107-3c11de2dfa52.png)
